armi/reactor/blocks.py

```diff
@@ -1737,18 +1685,6 @@ def getBoronMassEnrich(self):
             return 0.0
         return b10 / total
 
-    def getPuMoles(self):
```

Comment:
Just a head's up. I see this used downstream in certain depletion codes.

Reply:
All handled 😃